src/extension.ts

```diff
@@ -378,6 +386,83 @@ export function activate(context: vscode.ExtensionContext) {
     )
   );
 
+  subscriptions.push(
+    vscode.commands.registerCommand(
+      "vscode-couchbase.queryContext",
+      () => {
+        fetchQueryContext(workbench, context);
+      }
+    )
+  );
+
+  // subscription to make sure query context status bar is only visible on sqlpp files
+  subscriptions.push(
+    vscode.window.onDidChangeActiveTextEditor(async (editor) => {
+      await handleQueryContextStatusbar(editor, workbench);
+    })
+  );
+  // Handle initial view of context status bar
+  let activeEditor = vscode.window.activeTextEditor;
+  handleQueryContextStatusbar(activeEditor, workbench);
+
+  subscriptions.push(
+    vscode.commands.registerCommand(
+      "vscode-couchbase.showFavoriteQueries",
```

Comment:
Can you please move all these Commands to Command folder.
And use them as Commands.showFavourigeQueries

Reply:
Moved all commands to the folder